docs/SAFETY-MACROS.md

```diff
@@ -345,11 +340,6 @@ Ensures `result == _OSSL_SUCCESS`, otherwise the function will `POSIX_BAIL` with
 Ensures `s2n_result_is_ok(result)`, otherwise the function will return `S2N_FAILURE`
 
 
-### POSIX_GUARD_POSIX(result)
-
-Ensures `(result) >= S2N_SUCCESS`, otherwise the function will return `S2N_FAILURE`
-
-
```

Comment:
is the Deprecation message missing form some of these and the `PTR...` ones?

Reply:
Nice catch, it definitely is missing from all the x_GUARD_y ones. I'll add that in a separate PR.